include/bitcoin/bitcoin/utility/random.hpp

```diff
@@ -29,18 +30,24 @@ namespace libbitcoin {
 /**
  * Generate a pseudo random number within the domain.
  * @return  The 64 bit number.
+ *
+ * DEPRECATED in v3
```

Comment:
See existing convention [here](https://github.com/libbitcoin/libbitcoin/blob/master/include/bitcoin/bitcoin/wallet/encrypted_keys.hpp#L72). No need to qualify the version since it's inherent in the branch. Note also that we don't use `BC_DEPRECATED` because it creates heavy build warnings.